requirements.txt

```diff
@@ -3,4 +3,5 @@ nose>=1.3.0
 coverage>=3.6
 pylru>=1.0.3
 tldextract>=1.2
-six
\ No newline at end of file
+six
+pyyaml
```

Comment:
`pyyaml` is no longer developed, and annoyingly the actively maintained version is called `ruamel.yaml`, so we should use that.